### PR TITLE
FIX: Shorter rep

### LINKED ISSUE #560: Make event warnings shorter
Is this warning really useful to the user? We could probably have a shorter version.

```
WARNING: Error invoking callback <bound method MyCanvas.on_draw of <Vispy canvas (wx backend) at 0x2b0037ef9d90>> for event: <DrawEvent blocked=False handled=False native=None region=None source=<Vispy canvas (wx backend) at 0x2b0037ef9d90> sources=[<Vispy canvas (wx backend) at 0x2b0037ef9d90>] type=draw>
```

### PR DESCRIPTION
**Pull request by Eric89GXL**
Closes #560.

Test code:
```
from vispy.app import Canvas


class MyCanvas(Canvas):
    def on_draw(self, event):
        raise RuntimeError

x = MyCanvas(show=True)
```
Changes this output (`master`):
```
WARNING: Error invoking callback <bound method MyCanvas.on_draw of <Vispy canvas (PyQt4 (qt) backend) at 0x32dd940>> for event: <DrawEvent blocked=False handled=False native=None region=None source=<Vispy canvas (PyQt4 (qt) backend) at 0x32dd940> sources=[<Vispy canvas (PyQt4 (qt) backend) at 0x32dd940>] type=draw>
WARNING: Error invoking callback <bound method MyCanvas.on_draw of <Vispy canvas (PyQt4 (qt) backend) at 0x32dd940>> repeat 2
```
Into this:
```
RuntimeError
ERROR: Invoking <bound method MyCanvas.on_draw of <Canvas (PyQt4) at 0x237abe0>> for DrawEvent
ERROR: Invoking <bound method MyCanvas.on_draw of <Canvas (PyQt4) at 0x237abe0>> repeat 2
```
Ready for review/merge from my end.

### CODE REVIEW

File: vispy/util/logs.py [hunk 1]
```diff
@@ -317,14 +317,13 @@ def _handle_exception(ignore_callback_errors, print_callback_errors, obj,
         if this_print == 'full':
             logger.log_exception()
             if exp_type == 'callback':
-                logger.warning("Error invoking callback %s for "
-                               "event: %s" % (cb, event))
+                logger.error("Invoking %s for %s" % (cb, event))
```
Comment (rossant) [inline vispy/util/logs.py]: why the change from warning to error?

Comment (Eric89GXL) [inline vispy/util/logs.py]: Because it's actually indicative of an error. I found it weird to have it say `WARNING: Error ...`, seems like a misnomer.

**Comment by rossant** [comment]
LGTM. Maybe @almarklein or @campagnola should merge?
